provision/swarm/provisioner_test.go

```diff
@@ -1095,6 +1095,12 @@ func (s *S) TestDeploy(c *check.C) {
 	err := app.CreateApp(a, s.user)
 	c.Assert(err, check.IsNil)
 	attached := s.attachRegister(c, s.clusterSrv, true, a)
+	tags := []string{}
+	s.clusterSrv.CustomHandler("/images/.*/push", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		c.Assert(r.URL.Path, check.Equals, "/images/"+fmt.Sprintf("registry.tsuru.io/tsuru/app-myapp")+"/push")
```

Comment:
No need for Sprintf() and concat here, just make it a single string.

Reply:
ops, fixed that